notebook.ipynb

```diff
@@ -2,7 +2,15 @@
  "cells": [
```

Comment:
<p>Users could benefit from knowing what the var_names <code>tas</code>,<code>zg</code>,<code>uas</code>,<code>vas</code> means, and it would be nice to link to a data dictionary somewhere (so that others can find different variables). I usually recommend <a href="https://docs.google.com/spreadsheets/d/1UUtoz6Ofyjlpx5LdqhKcwHFz2SGoTQV2_yekHyMfL9Y/edit" target="_blank">https://docs.google.com/spreadsheets/d/1UUtoz6Ofyjlpx5LdqhKcwHFz2SGoTQV2_yekHyMfL9Y/edit</a> for CMIP6, but there might be a similar one for ERA5.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Added an explanation to variable mapping used, and the different options available in this implementation.</p>